components/Profile/ProfileBody.js

```diff
@@ -13,17 +13,25 @@ export default class ProfileBody extends Component {
   constructor(props) {
     super(props);
 
-    let routes = this.props.profile.roles === 'supporter' ? [
-      { key: 'details', title: 'Details' }
-    ] : [
-      { key: 'campaigns', title: 'Campaigns' },
-      { key: 'details', title: 'Details' }
-    ]
```

Comment:
The deleted version of this block is the correct version, the new version will cause a duplication of the location tab